### ISSUE TITLE
What should we do when we recieve an ACK for a message we don't understand?

### ISSUE THREAD

**Issue by cakekindel** [clov-coffee/kwap#76]

**Comment by cakekindel** [comment]
it would probably be most correct to respond with reset

**Comment by chrysn** [comment]
No, RST are only sent to respond do CON and maybe NON, quoting RFC752:

> Rejecting an Acknowledgement or Reset message (including the case where the
   Acknowledgement carries a request or a code with a reserved class, or
   the Reset message is not Empty) is effected by silently ignoring it.
   More generally, recipients of Acknowledgement and Reset messages MUST
   NOT respond with either Acknowledgement or Reset messages.
